logger.go

```diff
@@ -221,6 +221,11 @@ type LoggerOptions struct {
 	// Color the output. On Windows, colored logs are only avaiable for io.Writers that
 	// are concretely instances of *os.File.
 	Color ColorOption
+
+	// A filter which is called with the log information and returns true if the value
+	// should actually be logged. This is useful when interacting with a system that
```

Comment:
If `true` I think we’re suppressing the log.

Reply:
Quite right. I've fixed the docs.